### PR TITLE
503 Fix description in the module openy_media_document

### PR DESCRIPTION
**Pull request by osab**
https://www.drupal.org/node/2885220#comment-12123569

## Steps for review

 - [ ] login as admin
 - [ ] go to the /admin/structure/media/manage/document/fields/media.document.field_media_in_librarys
 - [ ] check for exist description for field_media_in_library and help text for it
 - [ ] go to the /admin/structure/media/manage/document/fields/media.document.field_media_tags
 - [ ] check for exist description field and help text for it

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by osab** [comment]
test this please

**Comment by Sanchiz** [comment]
ok to test

File: modules/openy_features/openy_media/modules/openy_media_document/config/install/field.field.media.document.field_media_in_library.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ field_name: field_media_in_library
 entity_type: media
 bundle: document
 label: 'Save to my media library'
-description: ''
+description: 'Select save or not to your media library'
```
Comment (ddrozdik) [inline modules/openy_features/openy_media/modules/openy_media_document/config/install/field.field.media.document.field_media_in_library.yml]: Let's say `Set checkbox if you want to save the image in your media library.`

**Review by ddrozdik — CHANGES_REQUESTED** [review]

File: modules/openy_features/openy_media/modules/openy_media_document/config/install/field.field.media.document.field_media_in_library.yml (hunk 1 above)
Comment (osab) [reply modules/openy_features/openy_media/modules/openy_media_document/config/install/field.field.media.document.field_media_in_library.yml]: ок

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1464 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1464 |
| Installation process | http://install.openy-dev.ffwua.com/build1464/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1464/

**Review by hamrant — APPROVED** [review]

**Review by ddrozdik — APPROVED** [review]

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1471/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #527 of commit 6a90d45c2ec8b9339f7a175ce623f6397119a374 automatically merged.
GitHub pull request #527 of commit 6a90d45c2ec8b9339f7a175ce623f6397119a374 automatically merged.
GitHub pull request #527 of commit 6a90d45c2ec8b9339f7a175ce623f6397119a374 automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace@8
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git3478230791865137004.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git5379167647736115630.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/527/merge^{commit} # timeout=10
 > git branch -a --contains f55b1abff891811f37e985769d4e921720929bc1 # timeout=10
 > git rev-parse remotes/origin/pr/527/merge^{commit} # timeout=10
Checking out Revision f55b1abff891811f37e985769d4e921720929bc1 (origin/pr/527/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f f55b1abff891811f37e985769d4e921720929bc1
 > git rev-list 75c0b2b8b88ac2f79471eb2de2df07c9bdd66f72 # timeout=10
First time build. Skipping changelog.
[workspace@8] $ /bin/sh -xe /tmp/hudson1405600365129805063.sh
+ rm -Rf /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm
+ cp -R ../vm /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm
+ git pull
Already up-to-date.
[workspace@8] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory3974971154918779451.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8 -e build_number=1471 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace@8] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace@8/vm/cibox/jobs/build.yml -i /tmp/inventory1488437095460797716.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace@8 -e build_number=1471 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace@8] $ ansible-playbook /var/www/build1471/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1471 -e drupal_folder=/var/www/build1471 -e site_url=http://openy-dev.ffwua.com/build1471 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
changed: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
skipping: [localhost]

TASK: [Remove settings.php] *************************************************** 
skipping: [localhost]

TASK: [Creating settings.php] ************************************************* 
skipping: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
skipping: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/default/files'})
skipping: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
skipping: [localhost]

TASK: [Creating drupal database] ********************************************** 
skipping: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
skipping: [localhost]

TASK: [Importing from local backup] ******************************************* 
skipping: [localhost]

TASK: [Remove local backup from build folder] ********************************* 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
skipping: [localhost]

TASK: [Clear Drupal cache] **************************************************** 
skipping: [localhost]

TASK: [Updating database] ***************************************************** 
skipping: [localhost]

TASK: [Apply pending entity schema updates] *********************************** 
skipping: [localhost]

TASK: [Download stage file proxy] ********************************************* 
skipping: [localhost]

TASK: [Enable stage file proxy] *********************************************** 
skipping: [localhost]

TASK: [Set configuration for stage file proxy] ******************************** 
skipping: [localhost]

TASK: [Set folder for stage file proxy] *************************************** 
skipping: [localhost]

TASK: [Updates listing] ******************************************************* 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
changed: [localhost]

TASK: [Installing drupal] ***************************************************** 
Build was aborted
Aborted by anonymous
```

**Pull request merged**